src/plugins/sync-timezone/host.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+module.exports = function hostSyncTimezone(host) {
+	host.onRequest('timezone', function() {
+		var htmlElems = document.getElementsByTagName('html');
+		if (htmlElems.length === 1 && htmlElems[0].hasAttribute('data-timezone')) {
+			try {
+				var timezone = JSON.parse(
+					htmlElems[0].getAttribute('data-timezone')
+				);
+				return {
+					id: timezone.id,
+					identifier: timezone.id,
```

Comment:
Why if you're not using it in the client?

Reply:
Good eye. Because this is going to [replace this](http://search.dev.d2l/source/xref/Lms/core/apploader/D2L.LP.AppLoader/Apps/Loaders/IFrame/V1/IFrameApp.js#262) which has `identifier` instead of `id`. Although seeing how gross this is, maybe I'll send it down as `identifier` instead of `id` on the HTML attribute just to keep everything happy and consistent.

Reply:
I've decided I'm going to make that change in LP, so this will just use `identifier` and that only. Will update.